doc/src/SUMMARY.md

```diff
@@ -2,6 +2,7 @@
 [Quick Installation](./install.md)
 [When to Use](./when-to-use.md)
 [Limitations and Workarounds](./limitations-and-workarounds.md)
+[For Developers](./for-developers.md)
```

Comment:
```suggestion
[For hevm Developers](./for-developers.md)
```

Reply:
It will also necessitate a name update to the file.

Reply:
Should be fixed now.